composer.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "moxie-leean/LeeanPlugin",
-  "description": "Barebones modular WordPress plugin",
+  "name": "moxie-leean/LeanWoocommerceApi",
```

Comment:
can you separate your package name by dashes `-` like`moxie-lean/woocomerce-api`